kafka_consumer/tests/conftest.py

```diff
@@ -85,6 +87,7 @@ def kafka_instance():
         'tags': ['optional:tag1'],
         'consumer_groups': {'my_consumer': {'marvel': [0]}},
         'broker_requests_batch_size': 1,
+        'api_version': os.environ.get('KAFKA_VERSION'),
```

Comment:
Not sure why `api_version` works but `kafka_client_api_version` doesn't work

Reply:
Looking into this more, `kafka_client_api_version` doesn't work when testing with `latest`, since `kafka_client_api_version` becomes `latest` rather than a real version number, and the code [tries to parse](https://github.com/DataDog/integrations-core/blob/master/kafka_consumer/datadog_checks/kafka_consumer/new_kafka_consumer.py#L47-L51) the version string to create the client, which results in the following error:

```
E   datadog_checks.dev.errors.RetryError: Result: False
E   Error: UnrecognizedBrokerVersion
E   Function: find_topics, Args: (), Kwargs: {}
```

I tried commenting out `api_version`, and the tests still pass, so maybe we can change it to `kafka_client_api_version` but instead of `latest` as an environment, we use `3.3.2` which corresponds to the version for latest.

Reply:
I ended up taking `api_version` out because I realized the "fix" was just a fluke. I added kafka and zk timeouts and that worked more reliably.